podcasts/Ratings/PodcastRatingViewModel.swift

```diff
@@ -30,7 +30,12 @@ class PodcastRatingViewModel: ObservableObject {
 
     /// Updates the rating for the podcast.
     ///
-    func update(podcast: Podcast?) {
+    func update(podcast: Podcast?, ignoringCache: Bool = false) {
+        // If we want to reload and ignore the cache, let's reset the state to waiting and reload
+        if ignoringCache, state == .done {
```

Comment:
Because the view controller calls the update many times, we use the `state` to avoid multiple calls. When the call finishes the state of the view model is set to `done`.
To be able to reload after the rating, I had to reset the state. I'm wondering though if we can eventually use just the `loading` and `done` state for this logic since the request uses the `useProtocolCachePolicy` policy by default:
- Return if the state is loading
- Call the API if it's different from loading setting the wanted cache policy

Reply:
Or if you already loading wait for the current state to be returned and return that?